examples/webgpu_compute_birds.html

```diff
@@ -0,0 +1,507 @@
+				container.appendChild( renderer.domElement );
+
+				// Initialize position, velocity, and phase values
+			
+				const positionArray = new Float32Array( BIRDS * 3 );
+				const velocityArray = new Float32Array( BIRDS * 3 );
+				const phaseArray = new Float32Array( BIRDS );
+
+				for ( let i = 0; i < BIRDS; i ++ ) {
+
+					const posX = Math.random() * BOUNDS - BOUNDS_HALF;
+					const posY = Math.random() * BOUNDS - BOUNDS_HALF;
+					const posZ = Math.random() * BOUNDS - BOUNDS_HALF;
+
+					positionArray[ i * 3 + 0 ] = posX;
+					positionArray[ i * 3 + 1 ] = posY;
+					positionArray[ i * 3 + 2 ] = posZ;
+
+					const velX = Math.random() - 0.5;
+					const velY = Math.random() - 0.5;
+					const velZ = Math.random() - 0.5;
+
+					velocityArray[ i * 3 + 0 ] = velX * 10;
+					velocityArray[ i * 3 + 1 ] = velY * 10;
+					velocityArray[ i * 3 + 2 ] = velZ * 10;
+
+					phaseArray[ i ] = 1;
+
+				}
+
+				// Create storage buffer attributes.
+			
+				const positionBufferAttribute = new THREE.StorageBufferAttribute( positionArray, 3 );
+				const velocityBufferAttribute = new THREE.StorageBufferAttribute( velocityArray, 3 );
+				const phaseBufferAttribute = new THREE.StorageBufferAttribute( phaseArray, 1 );
+
+				// Labels applied to storage nodes and uniform nodes are reflected within the shader output,
+				// and are useful for debugging purposes.
+
+				// Access storage buffer attribute data from within shaders with a StorageNode.
+			
+				const positionStorage = storage( positionBufferAttribute, 'vec3', positionBufferAttribute.count ).label( 'positionStorage' );
+				const velocityStorage = storage( velocityBufferAttribute, 'vec3', velocityBufferAttribute.count ).label( 'velocityStorage' );
+				const phaseStorage = storage( phaseBufferAttribute, 'float', phaseBufferAttribute.count ).label( 'phaseStorage' );
+			
+				// Create read-only storage nodes. Storage nodes can only be accessed outside of compute shaders in a read-only state.
+			
+				const positionRead = storageObject( positionBufferAttribute, 'vec3', positionBufferAttribute.count ).toReadOnly();
+				const velocityRead = storageObject( velocityBufferAttribute, 'vec3', velocityBufferAttribute.count ).toReadOnly();
+				const phaseRead = storageObject( phaseBufferAttribute, 'float', phaseBufferAttribute.count ).toReadOnly();
+
+				// Define Uniforms. Uniforms only need to be defined once rather than per shader.
+			
+				effectController = {
+					separation: uniform( 20.0 ).label( 'separation' ),
+					alignment: uniform( 20.0 ).label( 'alignment' ),
+					cohesion: uniform( 20.0 ).label( 'cohesion' ),
+					freedom: uniform( 0.75 ).label( 'freedom' ),
+					now: uniform( 0.0 ),
+					deltaTime: uniform( 0.0 ).label( 'deltaTime' ),
+					predator: uniform( new THREE.Vector3() ).label( 'predator' ),
+				};
+
+				// Create geometry
+			
+				const birdGeometry = new BirdGeometry();
+				const birdMaterial = new THREE.MeshStandardNodeMaterial();
```

Comment:
Sidenote: It seems the example produces the warning:

> AttributeNode.js:92 AttributeNode: Vertex attribute "normal" not found on geometry.

How about you create the bird material like so for now:
```js
const birdMaterial = new THREE.NodeMaterial();
birdMaterial.normals = false;
```